website/docs/guides/guides-upgrade.md

```diff
@@ -45,77 +45,124 @@ If you executed the previous step already on your server, then obviously nothing
+This is also available at `http(s)://<your-domain>/index.php?mod=install&action=db`.
 
 ### Test your installation
 
 Now is the time to look if everything is as expected.
 If yes, then you are good to unlock the installation.
 If not then either see what broke in the various logs or go to Rollback.
 
-## Rollback (if stuff doesn't work)
+### Unlock installation
+
+If you locked your installation, you  can now unlock it again.
+"Admin-Page" -> "Common Settings" -> "Lock LanSuite page".
+
+## Rollback (if things doesn't work)
 
-In case your upgrade failed terribly, and you need to return to your old state the following needs to be done:
+In case your upgrade failed, and you need to return to your old state the following needs to be done:
 
 ### Import DB backup
 
-You have to reimport the database backup taken before. How this is done varies in what tools you have available.
-The command line call for this would be `mysql -u<ls_user> -p < exportfile.sql`
+You have to reimport the database backup taken before.
+How this is done varies in what tools you have available.
+The command line call for this would be `mysql -u<ls_user> -p < exportfile.sql`.
 
 ### Restore folder backup
 
 Next step is to restore the folder backup taken.
-First clean up (`rm -rf <failed_update_folder>`) or move away (` mv <failed_update_folder> <anyothername>`) the failed installation.
+First clean up (`rm -rf <failed_update_folder>`) or move away (`mv <failed_update_folder> <anyothername>`) the failed installation.
 Then restore the original installation from your backup.
 Either by extracting (`gunzip ls_backup.tar.gz && tar xvf ls_backup.tar`) or moving back the copied folder (`mv <ls_folder>.backup <ls_folder>`).
 Or: Re-uploading your backup, if that was the way you went.
 
-### Unlock installation
+## Pitfalls and Known Bugs
 
-As the backup was taken at a point where the installation was locked, it may still be required to unlock it
-This would be done by reverting whatever was done to facilitate the original lock
+* Please ensure that export and import of Database images use the same character encoding. Using the same client on the same system should ensure this, but be cautious if dump and import are done on different systems/clients.
+* The git `master` branch is not usable without pulling in additional resources via composer. You must do this first to obtain a runnable installation!
 
-## Pitfalls and Known Bugs
+## Upgrade from LanSuite v4.2 to v5.0
+
+### Configuration: `database.sqlmode`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+sqlmode = "NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+[...]
+```
+
+### Configuration: `database.charset`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+charset = "utf8mb4"
+[...]
+```
+
+### Fonts in `ext_inc/pdf_fonts`
+
+Delete the following files
```

Comment:
I feel that we can provide a small script here to skip most if not all of the manual work

Reply:
We could provide one. I personally would love to see that we don't make it a requirement for this PR to get merged. Mainly, because I believe we first should make it work and our docs are not good. Manual work works at first and moves us forward vs. providing everything in a perfect shape.

Reply:
It's not a requirement, but I feel that this could be done easily and would eliminate the chance of human error